File: python/bee1005.py
notaA = round(float(input()), 1)
notaB = round(float(input()), 1)
if 0 <= notaA <= 10 and 0 <= notaB <= 10:
    media = ((notaA * 3.5) + (notaB * 7.5))/11
    print(f'MEDIA = {media:.5f}')

File: java/bee1005.java
import java.util.Scanner;

public class bee1005 {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);

        double nota1 = input.nextDouble();
        double nota2 = input.nextDouble();

        double media = ((nota1 * 3.5) + (nota2 * 7.5)) / 11.0;

        System.out.printf("MEDIA = %.5f\n", media);

        input.close();
    }
}
